aries-site/src/pages/foundation/accessibility.mdx

```diff
@@ -56,20 +50,22 @@ For guidance on setting up your development environment, see [Developer Guidance
+- **Try it yourself.** When testing your products manually by yourself, it is important to simulate the experience of a person with a disability to the best of your capabilities in order to build empathy. For example:
   - Turn your monitor off when using a screen reader. 
   - Avoid any reliance on your mouse. 
   - Use [technologies to aid in simulating disabilities](#disability-simulator-tools) such as color blindness, dyslexia and tunnel vision. 
-4. **Observe through real people.** The best way to understand the problems that users with disabilities face is to see how they navigate in their daily lives. Break down assumptions by learning about disabilities outside of the context of the web, too.
-5. **Conduct testing with real users.** Automated testing tools only catch approximately 30-50% of accessibility violations. Manual testing is essential to thoroughly conducting an accessibility test on a product. Test with a variety of users with disabilities to ensure full coverage. [Bill Tipton](https://hpe.sharepoint.com/sites/f5/cto/office/Accessibility/Pages/bill-tipton-biography.aspx), from the Product Accessibility Office, is an excellent internal resource for usability testing. Bill has used a screen reader in his daily life since 1999 when he lost his eyesight. Please email him or [Bill Bettega](https://hpe.sharepoint.com/sites/F5/CTO/Office/Accessibility/Pages/bill-bettega-biography.aspx) with any of your product testing requests and they will respond when they are able to assist.
-6. **Be an active advocate.** Testing does not end at the final report of accessibility violations of a product. To further advance accessibility, after testing, you should remain an ally and advocate for the disabled community. Amplifying all voices will lend a hand to making the internet more inclusive.
+- **Observe real people.** The best way to understand the problems that users with disabilities face is to see how they navigate in their daily lives. Break down assumptions by learning about disabilities outside of the context of the web, too.
+- **Test with real users.** Automated testing tools only catch approximately 30-50% of accessibility violations. Manual testing is essential to thoroughly conducting an accessibility test on a product. Test with a variety of users with disabilities to ensure full coverage. [Bill Tipton](https://hpe.sharepoint.com/sites/f5/cto/office/Accessibility/Pages/bill-tipton-biography.aspx), from the Product Accessibility Office, is an excellent internal resource for usability testing. Bill has used a screen reader in his daily life since 1999 when he lost his eyesight. Please email him or [Bill Bettega](https://hpe.sharepoint.com/sites/F5/CTO/Office/Accessibility/Pages/bill-bettega-biography.aspx) with any of your product testing questions and they will respond at the earliest convenience.
```

Comment:
```suggestion
- **Test with real users.** Automated testing tools only catch approximately 30-50% of accessibility violations. Manual testing is essential to thoroughly conducting an accessibility test on a product. Test with a variety of users with disabilities to ensure full coverage. [Bill Tipton](https://hpe.sharepoint.com/sites/f5/cto/office/Accessibility/Pages/bill-tipton-biography.aspx), from the Product Accessibility Office, is an excellent internal resource for usability testing. Bill has used a screen reader in his daily life since 1999 when he lost his eyesight. Please email him at [Bill Bettega](https://hpe.sharepoint.com/sites/F5/CTO/Office/Accessibility/Pages/bill-bettega-biography.aspx) with any of your product testing questions and they will respond at the earliest convenience.
```

Reply:
didn't realize it was two separate people so ignore my comment